### DISCUSSION TITLE
VictoryPie radius prop event mutations are not animated

### DISCUSSION THREAD

**Discussion by brendanmckeown** [FormidableLabs/victory#2103] [General]
# Bugs and Questions

### Checklist

- [x] This is not a `victory-native` specific issue. (Issues that only appear in `victory-native` should be opened [here](https://github.com/FormidableLabs/victory-native/issues/new))

- [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

- [x] I am using the latest version of Victory

- [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

When the `radius` prop for a `VictoryPie` is mutated using the standard single component events, the mutations are not animated, even if the `animate` prop is defined on the component.

### Reproduction

See https://codesandbox.io/s/old-moon-um2u3?file=/index.js for an example.

**Comment by FRizzonelli** [comment]
I can confirm the issue, is there any update on this?

**Comment by boygirl** [comment]
Apologies for the delay. None of the event system mutations animate anywhere in Victory. In order to support this change, we would need to move the "level" of the animation from components like `VictoryPie` to primitive components like `Slice`. This is a potentially very intrusive change, and one that we would need to evaulate for perfomance implications as well as correctness with coordinating transitioning elements.
